### PR TITLE
Fix/114373 para Release

### PR DESCRIPTION
**Pull request by julianocm**
AB#114373

### CODE REVIEW

**Comment by luizhpriotto** [comment]
❌ There was a problem linking to Azure Boards work item(s):
- 114373

Please check the IDs and try again using the AB# syntax. [Learn more](https://aka.ms/ablinks)

File: sme.gclass.application/Queries/ConectaFormacao/ListagemDetalhamentoFormacao/ListagemDetalhamentoFormacaoQueryHandler.cs [hunk 1]
```diff
@@ -41,8 +41,10 @@ public async Task<IEnumerable<FormacaoDetalhaDTO>> Handle(ListagemDetalhamentoFo
 
             var professoresRegentesTutores = professoresRegentes.Union(professoresTutores);
 
-            var cpfsDosProfessores = await repositorioProfessorEol.ObterCpfNomeCompletoPorRegistroFuncional(professoresRegentesTutores.Select(s => s.Rf).Distinct().ToArray());
-            var usuariosProfessores = await repositorioUsuario.ObterUsuariosGooglePorCodigos(professoresRegentesTutores.Select(t => long.Parse(t.Rf)).Distinct().ToArray(), new[] { 2, 3 });
+            var rfs = professoresRegentesTutores.Where(t => !string.IsNullOrEmpty(t.Rf)).Select(t => t.Rf).Distinct().ToArray();
+
+            var cpfsDosProfessores = await repositorioProfessorEol.ObterCpfNomeCompletoPorRegistroFuncional(rfs);
+            var usuariosProfessores = await repositorioUsuario.ObterUsuariosGooglePorCodigos(rfs.Select(t => long.Parse(t)).ToArray(), new[] { 2, 3 });
```
Comment (viniciusnyari) [inline sme.gclass.application/Queries/ConectaFormacao/ListagemDetalhamentoFormacao/ListagemDetalhamentoFormacaoQueryHandler.cs:47]: Coloca esses caras como enum

**Review by viniciusnyari — CHANGES_REQUESTED** [review]

**Comment by luizhpriotto** [comment]
❌ There was a problem linking to Azure Boards work item(s):
- 114373

Please check the IDs and try again using the AB# syntax. [Learn more](https://aka.ms/ablinks)

**Review by viniciusnyari — APPROVED** [review]

**Comment by luizhpriotto** [comment]
❌ There was a problem linking to Azure Boards work item(s):
- 114373

Please check the IDs and try again using the AB# syntax. [Learn more](https://aka.ms/ablinks)

**Pull request merged**